Tests/AppStoreConnectClientTests/AppStoreConnectClientTests.swift

```diff
@@ -200,14 +200,14 @@ final class AppStoreConnectClientTests: XCTestCase {
         let build = try await client.fetchBuilds(
             for: app,
             with: BuildsQuery.init(
-                sort: .init(arrayLiteral: ._hyphen_version),
-                fields: .init(arrayLiteral: .version, .minOsVersion, .uploadedDate)
+                sort: .hyphenVersion,
+                fields: [.version, .minOsVersion, .uploadedDate]
             )
         )
-        XCTAssertEqual(build.first?.id, expectedBuilds.first?.id)
-        XCTAssertEqual(build.first?.version, expectedBuilds.first?.version)
-        XCTAssertEqual(build.first?.uploadedDate, expectedBuilds.first?.uploadedDate)
-        XCTAssertEqual(build.first?.minOsVersion, expectedBuilds.first?.minOsVersion)
+        XCTAssertEqual(build?.first?.id, expectedBuilds.first?.id)
```

Comment:
Result must be comparable and then compare object

Reply:
Changed to XCTAssertEqual(build, expectedBuilds)